FAVCS-30 : Media browser plus PATCH, Added Plupload Integration, Hook…

mnakov opened the pull request:
Patched Media Browser plus module
Installed Plupload integration module 
Enabled Media Bulk Upload module

vzlatev commented on drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install:
```diff
@@ -916,3 +916,12 @@ function favrskov_helper_update_7049() {
 }
+
+/**
+ * Implements hook_update
```
This should be changed Implements hook_update_N().

vzlatev commented on drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install:
```diff
@@ -916,3 +916,12 @@ function favrskov_helper_update_7049() {
+ */
+function favrskov_helper_update_7050() {
+  module_enable(array('media_bulk_upload'));
+
```
No need for blank row here.

mnakov commented:
### https://ffwagency.atlassian.net/projects/FAVCS/queues/custom/351/FAVCS-30
  
### Changes
- Fixed problem with deleting multiple files at a time
  
### Technical Details
- Added hook_update for enabling media_bulk_upload module
- Installed dependency module for media_bulk_upload: Plupload integration module
  
### Affected Pages
- admin/content/file
- admin/content/file/list
  
### Key Affected Code
- media_browser_plus.module
- favrskov_helper.install
  
### Key Functionality in custom code
- Applied patch to fix redirecting bug in Media Browser Plus module
  
### References
- https://www.drupal.org/files/issues/media_browser_plus-views_bulk_actions_broken-2209833-2-7.patch
  
### Notices
- N/A
